src/DataSet/ObjectDataSet.php

```diff
@@ -299,4 +305,15 @@ public function getAttributeTranslator(): ?AttributeTranslatorInterface
     {
         return $this->parser->getAttributeTranslator();
     }
+
+    public function getValidationPropertyLabels(): array
```

Comment:
How about shorter name?

```suggestion
    public function getPropertyLabels(): array
```